tensorflow/compiler/mlir/tosa/transforms/passes.h

```diff
@@ -68,6 +68,8 @@ std::unique_ptr<OperationPass<func::FuncOp>> createLowerComplexTypesPass();
 std::unique_ptr<OperationPass<func::FuncOp>> createStripFunctionMetadataPass();
 std::unique_ptr<OperationPass<func::FuncOp>> createStripQuantTypesPass();
 std::unique_ptr<OperationPass<func::FuncOp>> createVerifyFullyConvertedPass();
+std::unique_ptr<OperationPass<ModuleOp>> createLegalizeTFLStatefulPass();
```

Comment:
Could we keep these sorted?

Reply:
I propose we keep this change in another patch since the existing order is not sorted. Then this patch will be logically cleaner.